### DISCUSSION TITLE
Wireless debugging allow manual pairing.

### DISCUSSION THREAD

**Discussion by rgrimjr40** [RikkaApps/Shizuku#557] [Ideas]
I have an LG v60. The device does not have an option to enable wireless debugging. It is a pain to connect my phone to a PC every time Ive to run Shizuku. Therefore, I saw an app that uses Shizuku called Wireless ADB Switch. I got exited thinking this would finally resolve my issue. However, there is not a feature to manually enter the connection details. Can you please add an option so I can paste the connection details to Shizuku and connect? Pretty please? This is a pain and hassle for me.

**Comment by rgrimjr40** [comment]
Is there a way to start Shizuku without me needing a computer? As I mentioned above my phone does not have a way to enable wireless debugging?

**Reply by shaggyskunk** [reply]
There's an app called "adb ⚡OTG" , that allows you to use adb commands without a computer. You do need to use Two Android phones though.

**Reply by rgrimjr40** [reply]
Cool thanks for the reply. I will take a look at it.
